db/controllers/news.controller.js

```diff
@@ -17,3 +22,16 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
```

Comment:
Please split your models and controllers out into the different tables/ data they're working with - for example articles-model, topics-model. It will make your project much nicer to navigate!

Reply:
This was fun ;)